nextcord/message.py

```diff
@@ -362,9 +372,23 @@ def to_dict(self) -> AttachmentPayload:
             result["content_type"] = self.content_type
         if self.description:
             result["description"] = self.description
+        if self.duration_secs:
+            result["duration_secs"] = self.duration_secs
+        if self._waveform:
+            result["waveform"] = self._waveform
         return result
 
-    @property
+    @utils.cached_property
+    def waveform(self) -> Optional[array[int]]:
+        """Optional[array[:class:`int`]]: The base64 decoded data representing a sampled waveform
+        (currently for voice messages).
+
+        .. versionadded:: 3.0
+        """
+        if self._waveform is not None:
+            return array("B", [int(b) for b in base64.b64decode(self._waveform)])
```

Comment:
```suggestion
            return array("B", base64.b64decode(self._waveform))
```

`base64.b64decode` returns `bytes` which when iterated return `int`